pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -60,12 +60,12 @@ def __post_init__(self):
     async def stream_output(self, *, debounce_by: float | None = 0.1) -> AsyncIterator[OutputDataT]:
         """Asynchronously stream the (validated) agent outputs."""
         async for response in self.stream_responses(debounce_by=debounce_by):
-            if self._final_result_event is not None:
+            if self._raw_stream_response.get_final_result_event() is not None:
```

Comment:
Is using a method here now adding any meaningful overhead? If not this is probably fine